refreshable/refreshable_validating.go

```diff
@@ -41,6 +27,22 @@ func (v *ValidatingRefreshable) LastValidateErr() error {
 // validatingFn successfully. This returns an error if the current value of the passed in Refreshable does not pass the
 // validatingFn or if the validatingFn or Refreshable are nil.
 func NewValidatingRefreshable(origRefreshable Refreshable, validatingFn func(interface{}) error) (*ValidatingRefreshable, error) {
+	mappingFn := func(i interface{}) (interface{}, error) {
+		if err := validatingFn(i); err != nil {
+			return nil, err
+		}
+		return nil, nil
+	}
+	return newValidatingRefreshable(origRefreshable, mappingFn, false)
+}
+
+// MapValidatingRefreshable is similar to NewValidatingRefreshable but allows for the function to return a mapping/mutation
```

Comment:
Can you also mention that the returned value is what is stored as the current value of the refreshable, rather than the current value of the original refreshable. It's a subtle nuance but seems important to call out for the docs.

Reply:
Updated, lmk what you think